### ISSUE TITLE
Issue installing OVA on vsphere essentials

### ISSUE THREAD

**Issue by bhundven** [vmware/photon-controller#136]
![screenshot from 2017-06-20 23-00-55](https://user-images.githubusercontent.com/1155179/27398099-8c85122a-566d-11e7-8b2e-67d34755a9ce.png)

On the third step to choose the cluster/datacenter/host to use for photon-controller, it crashes with the attached OVA error. I'm not sure if this is a setup issue or really an issue with the OVA.

ESXi: 6.5.0/5310538
VCSA: 6.5.0/5705665

**Comment by MylesGray** [comment]
Same here, VCSA 6.5.0/5705665 and ESXi 6.5.0/5310538.

Have tried deploying to Resource Pool, Cluster and directly to a host. From what I can see PIIX4 is a PCI to IDE controller?

**Comment by It4lik** [comment]
Hi, 
Did you read the doc ? 
Don't know if it's related to your error but your ESXi hosts should NOT be managed by any VCSA.
And you should have your ESXi running with a specific patch : ESXi650-201701001 (build number 4887370). 
Both of these informations are in [the Assumptions section of the doc](https://github.com/vmware/photon-controller/wiki/Quick-Start-Guide#assumptions)

**Comment by bhundven** [comment]
@It4lik If I have to run a specific patch of esxi 6.5.0, and I cannot reuse the existing infrastructure I already use, then it makes little sense to use this project. This has been an absolute waste of time.

**Comment by It4lik** [comment]
@bhundven yes, I understand.  I think the whole point of Photon Controller is to dedicate some hardware to it, and that means dedicate hardware to cloud-native apps. This allows you to have a 
 scale-out IaaS platform dedicated to cloud-native apps. Some applications are natively packaged (and so are Kubernetes or Harbor at the moment, for example). You can give direct access to any of your "clients" : admins, devs, etc, in a isolated tenant with resources allocated to this tenant. This promotes autonomy, speed and flexibility. 
I think that if you want to keep you current infrastructure, you should get interested in VIC project (especially VIC engine which make container hosts (VCH) first-class citizen in vSphere).
